### DISCUSSION TITLE
Case Law database should show last update date

### DISCUSSION THREAD

**Discussion by rbpasker** [freelawproject/courtlistener#4578] [Ideas]
To support proper academic citations, this page should show the date of last update

https://www.courtlistener.com/opinion/

see: https://github.com/freelawproject/courtlistener/issues/4573

**Comment by mlissner** [comment]
Do you mean the last time we got new content in the system? Not sure what you have in mind.

**Reply by rbpasker** [reply]
"In Chicago Manual of Style, when citing an online database, the “date last updated” is the most recent date provided by the database that indicates when the content or resource was last revised or modified. If no specific “last updated” date is listed, you should use the date you accessed the material.

Here is an example of how it might look in a full note:

Format:
Author’s First Name Last Name, Title of Work, Last modified [or updated] Month Day, Year, URL.

If there is no modification date, just include the access date:

Example:
John Doe, Database Title, accessed October 15, 2024, URL."

**Reply by mlissner** [reply]
Hm, we update the database constantly. I think the date last updated would always be today, except perhaps on weekends.

**Reply by rbpasker** [reply]
The proposal is to post the date the database was last updated on the website. I'm hoping the response will be: Yes we're going to do this, or no we're not going to do this.  

'the “date last updated” is the most recent date provided by the database'

**Comment by rbpasker** [comment]
The website still has placeholders for [date] and [url]

if you're not going to fix it, at least fix the website

https://case.law/about/#data-citation

<img width="681" alt="image" src="https://github.com/user-attachments/assets/16601acf-1d95-458c-a3e2-370435f9deec">
